config/application.rb

```diff
@@ -60,6 +60,13 @@ class Application < Rails::Application
     # Version of your assets, change this if you want to expire all your assets
     config.assets.version = '1.0'
 
+    # Set cache headers
+    config.static_cache_control = "public, max-age=31536000"
```

Comment:
http://edgeguides.rubyonrails.org/5_0_release_notes.html#railties-deprecations

Shall we re-write as following?
`config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=31536000' }`